applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -656,7 +722,7 @@ To enable the Matter smart plug functionality, run the following command:
 
       .. code-block:: console
 
-         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_Wi-Fi_NRF700X=y -DCONFIG_CHIP_Wi-Fi=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
```

Comment:
Please verify all configs using WIFI in this readme, as I think that you replaced all WIFI -> Wi-Fi occurrences what broke multiple build commands.